models/households-model.js

```diff
@@ -14,7 +14,6 @@ const insert = (newHousehold) => {
 
 const update = (id, updates) => {
   return db('households')
-    .where({ id })
```

Comment:
Why is this change here? Doesn't it need an id to specify which household?

Reply:
I'm not entirly sure